.github/dependabot.yml

```diff
@@ -32,6 +32,11 @@ updates:
     ignore:
       - dependency-name: "org.slf4j.slf4j-api"
       - dependency-name: "org.openapi.generator"
+    # https://til.simonwillison.net/github/dependabot-python-setup
+    groups:
+      python-packages:
+        patterns:
+          - '*'
```

Comment:
I like the idea, but I rarely merge those updates so I'm not sure how often we do some selective merges, where we update some and packages and leave others. Let's hear from @mobuchowski or @pawel-big-lebowski here.

Also, shouldn't we at least split those PRs into multiple groups f.e. java / python / rust ? Or if they are to be grouped together, maybe we should not name the group `python-packages`?

Reply:
Fixed name

Reply:
My only concern are versions we don't want to bump as CI is failing on new version. Like Hadoop map reduce on the screenshot attached from 2.10.2 -> 3.4.0. But I assume this should be solved telling dependabot to ignore certain packages.

Reply:
If this change introduces more issues than is solves, then I'm okay with closing this PR

Reply:
I am happy to test this in action.

Reply:
Let's wait a day or two in case someone comes with strong `no`. Otherwise, we merge it.

Reply:
Very easy to reverse that change if we don't like - let's do it.